killbill-integration-tests/plugins/killbill-email-notifications/test_email_notifications.rb

```diff
@@ -86,16 +113,13 @@ def setup
 
     end
 
-
     def teardown
-      teardown_base
-      #@smtp_server.shutdown_and_wait_for_completion
+      teardown_plugin_base(PLUGIN_KEY)
+      @smtp_server.shutdown_and_wait_for_completion
```

Comment:
If the start sequence fails, this can be `nil`:

```
===============================================================================
Error: test_basic(KillBillIntegrationTests::TestEmailNotification)
  NoMethodError: undefined method `shutdown_and_wait_for_completion' for nil:NilClass
killbill-integration-tests/killbill-integration-tests/plugins/killbill-email-notifications/test_email_notifications.rb:118:in `teardown'
===============================================================================
```